File: Java/Conditional Statements/Exercise/ConditionsExercise/src/BonusScore.java
import java.util.Scanner;

public class BonusScore {

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

        int initPoints = Integer.parseInt(input.nextLine());
        double bonusPoints = 0;

        if(initPoints <= 100){
            bonusPoints += 5;

        } else if (initPoints > 100 && initPoints <= 1000){
            bonusPoints += initPoints * 0.2;

        }else {
            bonusPoints += initPoints * 0.1;
        }

        if(initPoints % 2 == 0){
            bonusPoints += 1;
        } else if (initPoints % 5 == 0){
            bonusPoints += 2;
        }

        double totalPoints = (double) initPoints + bonusPoints;

        System.out.printf("%.1f\n%.1f", bonusPoints, totalPoints);
    }

}

File: Python/Basics/Conditional Statements/Exercises/Bonus Score.py
initPoints = int(input())

bonusPoints = 0
totalPoints = initPoints

if totalPoints <= 100:
    bonusPoints = 5

elif 100 < totalPoints <= 1000:
    bonusPoints += initPoints * 0.2

else:
    bonusPoints += initPoints * 0.1

if totalPoints % 2 == 0:
    bonusPoints += 1

if totalPoints % 10 == 5:
    bonusPoints += 2

totalPoints += bonusPoints

print(bonusPoints)
print(totalPoints)